lzy/allocator/src/main/java/ai/lzy/allocator/services/AllocatorPrivateService.java

```diff
@@ -123,13 +124,17 @@ public void register(RegisterRequest request, StreamObserver<RegisterResponse> r
                             return Status.INTERNAL.withDescription("Cannot get endpoints of vm");
                         }
 
+                        var meta = allocationContext.vmDao().getAllocatorMeta(vm.vmId(), transaction);
```

Comment:
`vm` (line 68) already contains meta

Reply:
But that meta doesn't contain my property, which appears in database on line 118, after collecting vm object in line 68.